Library/Homebrew/cmd/audit.rb

```diff
@@ -1235,3 +1232,31 @@ def problem(text)
     @problems << text
   end
 end
+
+# Debugging support
+
+def instrument_module_with_method_timers(the_module)
```

Comment:
It would be better to turn it to a method of `FormulaAuditor` like what `brew doctor` does.